### ISSUE TITLE
[第3章]datasetの作成内の色の平均値と標準偏差に関して

### ISSUE THREAD

**Issue by kirohirahanoshi** [YutaroOgawa/pytorch_advanced#60]
dataset作成の最終コードで動作の確認をされております。
このコード内に色の平均値と標準偏差が、mean = (0.485, 0.456, 0.406), std = (0.229, 0.224, 0.225)と記載されています。
こちらに関しては1章の色情報の規格化の項から来た値と考えましたが、合っていますでしょうか?
その場合、1章ではこれらの数値はILSVRC2012のデータセットに基づいた値と記載されておりますが、この数値をどのデータセットに対しても一般的に使用することは問題ないと解釈して良いということでしょうか？

**Comment by YutaroOgawa** [comment]
@kirohirahanoshi さま

ご質問いただき、ありがとうございます。

まず、0からネットワークを訓練させる場合、データセットの色の平均と標準偏差は、「自分が訓練させたい画像をまとめたデータセット」の平均値と標準偏差を使用します。
その平均値と標準偏差で実際に訓練するので。

続いて、本書の第1章の転移学習・ファインチューニングのような、訓練済みのネットワークを利用する場合は、その訓練時の平均と標準偏差を使用します。

今回の平均と標準偏差の値は、以下ページ
https://pytorch.org/docs/stable/torchvision/models.html
の
>All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. You can use the following transform to normalize:

から来ています。

そのため、0からネットワークを訓練させる場合は手持ちの訓練データセットでの標準化に使用する平均値、標準偏差を使用。
転移学習やファインチューニングは、訓練時の設定を使用、となります。

今後も、不明点等ございましたら、気軽に質問いただければ幸いです。
どうぞよろしくお願い致します。
